packages/carequality-sdk/jest.config.ts

```diff
@@ -0,0 +1,18 @@
+import type { Config } from "@jest/types";
+
+const isE2E = process.env["E2E"] === "true";
```

Comment:
when / where do we set this to true? In our package.json runscripts and CI?

Reply:
see `package.json` files - or search for `E2E=` on the codebase